plugins/inputs/mysql/mysql.go

```diff
@@ -230,6 +230,11 @@ const (
         SELECT NAME, COUNT
         FROM information_schema.INNODB_METRICS
         WHERE status='enabled'
+    `
+	innoDBMetricsQueryMariadb = `
+        SELECT NAME, COUNT
+        FROM information_schema.INNODB_METRICS
+        WHERE ENABLED=1
```

Comment:
@MarcHagen thanks for this, and it looks like folks have tested it on mariadb versions 10.5 and newer. 

What will happen to a user on mariadb versions older than 10.5? I ask since it looks like 10.2, 10.3, and 10.4 have had releases this year already.

Reply:
@powersj Below 10.5 that query will indeed fail.

Then we need to change the `if m.MariadbDialect` to check for version with something like `SELECT version()` that we then can regex `10\.[1-4].*` ?

What is your opinion on that?

Reply:
Or something like this, `EXECUTE IMMEDIATE` is instroduced in MariaDB 10.2.3  
```
EXECUTE IMMEDIATE CONCAT("
SELECT NAME, COUNT
FROM information_schema.INNODB_METRICS
WHERE ", IF(version() REGEXP '10\.[1-4].*',"status='enabled'", "ENABLED=1"), "
");
```

Reply:
That sounds good, my aim is to avoid breaking existing users on older, still supported versions. As such the version check seems to make the most sense.

If you could give that a try and let me know what version you are able to try it on, that would be great!

Reply:
I've tested the query against 10.1 (expected fail, is EOL) 10.2, 10.5 and 10.6 servers.
From 10.2 up to 10.6 all works fine. Seems to me a good solution for the version gap between <10.5 and >10.5